distributed/queues.py

```diff
@@ -271,13 +270,7 @@ def close(self):
             self.client._send_to_scheduler({"op": "queue_release", "name": self.name})
 
     def __getstate__(self):
-        return (self.name, self.client.scheduler.address)
+        return self.name
 
     def __setstate__(self, state):
-        name, address = state
-        try:
-            client = get_client(address)
-            assert client.scheduler.address == address
-        except (AttributeError, AssertionError):
-            client = Client(address, set_as_default=False)
-        self.__init__(name=name, client=client)
```

Comment:
I dropped some complexity here. This is reflected by the test in https://github.com/dask/distributed/pull/7564/files#r1113125731

We basically no longer bind the future to the address. This is not required and I can roll this back. I tried removing future serialization entirely at one point but was stopped by the publish plugin which actually reassembles the original collection